sapl/base/views.py

```diff
@@ -1911,8 +1914,11 @@ def form_valid(self, form):
         user = form.save(commit=False)
         data = form.cleaned_data
 
-        # new_user.first_name = data['firstname']
-        # new_user.last_name = data['lastname']
+        if data['first_name'] and data['first_name'] != user.first_name:
```

Comment:
Não é assim que se faz para verificar se o `first_name` está no dic `data`! 
```suggestion
        if 'first_name' in data and data['first_name'] != user.first_name:
```